seeding/seeding-integration-test/all/01/06_technical-user.json

```
@@ -0,0 +1,13 @@
+{
+    "entityName": "TechnicalUser",
+    "entities": [
+        {
+            "id": 7777777,
+            "username": "technical",
```

@tikle-cap Pls change to something descriptive, e. g.
```suggestion
            "username": "cron-runner",
```